infra/toynet/terraform/devUsers.tf

```diff
@@ -6,7 +6,7 @@ variable "ami_id" {
 
 variable "key_name" {
   type = string
-  default = "toynet-2020"
+  default = "toynet-key-theo-2020"
```

Comment:
I was thinking some more about the `toynet-intern-2020` key... maybe it's okay since production would have different keys, and this way I can trouble shoot an intern's machine with the same key. What do you think?

Reply:
Or.... maybe only I hold this key, and we set them up with a login?

Reply:
I believe best practice would be for everyone to have their own key. But really I could make an argument for both options.

Reply:
I suppose there will be no ssh just screen share on each dev machine. That works! A followup task:

Take a look at what I do with `instance_names` in this file and see if you can get it so that given a list of intern names, we can construct an instance for them AND generate a key for them. I also wonder if we are able to give them the key automagically once it's generated? 🤔

Reply:
I was just wondering the same thing! I’ll look into this as well.